### PR TITLE
added printAll, printNext, printError, ignoreError methods

### PR DESCRIPTION
**Pull request by larryonoff**

### CODE REVIEW

File: Source/Signal.swift [hunk 1]
```diff
@@ -25,3 +25,21 @@ public func filterMap<T, U, E>(transform: T -> U?)(signal: Signal<T, E>) -> Sign
...
+
+/// Returns a signal that drops any errors
+public func ignoreError<T, E>(signal: Signal<T, E>) -> Signal<T, NoError> {
+    return Signal { observer in
+        return signal.observe(Signal.Observer { event in
+            switch event {
+            case let .Next(value):
+                sendNext(observer, value.value)
+            case let .Error(error):
+                break
```
Comment (neilpa) [inline Source/Signal.swift]: This should send completed so this doesn't become an infinite signal.

**Comment by neilpa** [comment]
:tada: Thanks, just one comment

File: Source/Signal.swift (hunk 1 above)
Comment (larryonoff) [inline Source/Signal.swift]: Do you mean on Error ?

Comment (neilpa) [inline Source/Signal.swift]: Yea, since `Error` is a terminal event we should replace it with `Completed`.

**Comment by larryonoff** [comment]
Hopefully fixed the comment.

Could you please create a new tag after merge.

**Comment by neilpa** [comment]
:+1: 

> Could you please create a new tag after merge.

Was already planning on it.

**Comment by larryonoff** [comment]
wait a minute for tag please. I'll create one more pull request. :)

**Comment by neilpa** [comment]
Too late, but tags are cheap and I can create another one.

**Pull request merged**